### PR TITLE
Layers to workflow

### PR DESCRIPTION
**Pull request by jdroenner**
- [x] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by jdroenner** [comment]
[Layer zeigen eine Information über ihre Gültigkeit.](https://trello.com/c/IYjLw7qr/1026-layer-zeigen-eine-information-%C3%BCber-ihre-g%C3%BCltigkeit)

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: services/src/handlers/layers.rs [hunk 1]
```diff
@@ -484,6 +489,51 @@ async fn layer_handler<C: Context>(
...
+        ("layer" = LayerCollectionId, description = "Layer id"),
+    ),
+    security(
+        ("session_token" = [])
+    )
+)]
+async fn layer_to_workflow_id_handler<C: Context>(
+    ctx: web::Data<C>,
+    path: web::Path<(DataProviderId, LayerId)>,
+) -> Result<impl Responder> {
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply services/src/handlers/layers.rs]: [d42dbb6](https://github.com/geo-engine/geoengine/pull/692/commits/d42dbb6cb0508ea353f4dcaa40d9ec91acda2ad3)

**Comment by ChristianBeilschmidt** [comment]
Please resolve the conflicts.

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by ChristianBeilschmidt** [comment]
bors retry

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by jdroenner** [comment]
bors retry

**Pull request merged**